buffalo/data/buffered_data.py

```diff
@@ -179,88 +179,88 @@ class BufferedDataStream(BufferedData):
     """
     def __init__(self):
         super().__init__()
-        self.major = {'rowwise': {}}
-        self.group = 'rowwise'
+        self.major = {"rowwise": {}}
+        self.group = "rowwise"
 
     def free(self, buf):
-        buf['indptr'] = None
-        buf['keys'] = None
+        buf["indptr"] = None
+        buf["keys"] = None
 
     def initialize(self, data):
         self.data = data
-        assert self.data.data_type == 'stream'
+        assert self.data.data_type == "stream"
         # 12 bytes(indptr8, keys4)
         limit = max(int(((self.data.opt.data.batch_mb * 1000 * 1000) / 12.)), 64)
         minimum_required_batch_size = 0
         lim = int(limit / 2)
-        G = 'rowwise'
+        G = "rowwise"
         group = data.get_group(G)
         header = data.get_header()
         m = self.major[G]
         if self.major[G]:
             self.free(self.major[G])
-        m['index'] = 0
-        m['limit'] = lim
-        m['start_x'] = 0
-        m['next_x'] = 0
-        m['max_x'] = header['num_users']
-        m['indptr'] = group['indptr'][::]
-        minimum_required_batch_size = max([m['indptr'][i] - m['indptr'][i - 1]
-                                           for i in range(1, len(m['indptr']))])
-        m['keys'] = np.zeros(shape=(lim,), dtype=np.int32)
+        m["index"] = 0
+        m["limit"] = lim
+        m["start_x"] = 0
+        m["next_x"] = 0
+        m["max_x"] = header["num_users"]
+        m["indptr"] = group["indptr"][::]
+        minimum_required_batch_size = max([m["indptr"][i] - m["indptr"][i - 1]
+                                           for i in range(1, len(m["indptr"]))])
```

Comment:
사소한건데 아래처럼 하면 좀더 간단할거 같습니다.

```suggestion
        minimum_required_batch_size = np.ediff1d(m["indptr"]).max()
```